flycheck.el

```diff
@@ -6079,7 +6084,17 @@ See URL `http://handlebarsjs.com/'."
           "Error: Parse error on line " line ":" (optional "\r") "\n"
           (zero-or-more not-newline) "\n" (zero-or-more not-newline) "\n"
           (message) line-end))
-  :modes (handlebars-mode handlebars-sgml-mode))
+  :modes (handlebars-mode handlebars-sgml-mode web-mode)
+  :predicate (lambda ()
+               (or (and (eq major-mode 'web-mode)
```

Comment:
Please turn the whole conditional into an `(if (eq major-mode 'web-mode) … t)`, since the checks below are really only relevant if the major mode is Web Mode.  Currently the entire conditional is a little hard to follow.